### DISCUSSION TITLE
如何提高表格识别精度

### DISCUSSION THREAD

**Discussion by adamzhg** [PaddlePaddle/PaddleOCR#14122] [Q&A]
使用默认参数进行表格识别，但效果不佳。
输入图片如下：
![2024-06-10-00-3-4-CRH1E-1075_1](https://github.com/user-attachments/assets/992b3311-1e0a-46bb-b261-59470bda7acb)
模型定义的参数如下：
det_model_dir：用的是ch_PP-OCRv4_det_server_infer
rec_model_dir：用的是ch_PP-OCRv4_rec_server_infer
table_model_dir：用的是ch_ppstructure_mobile_v2.0_SLANet_infer
但是识别效果不理想：
![1730264470479](https://github.com/user-attachments/assets/e3decfba-6e29-4c7c-b800-45bc37c8e22e)
[[1, 106, 3499, 2259]_0.xlsx](https://github.com/user-attachments/files/17566298/1.106.3499.2259._0.xlsx)
表格识别有很多错误和窜位。

因为图片本身分辨率比较大，尝试提高table_max_len参数到1024、2048等，但效果反而更差。

请问针对这类较复杂表格，除了重新训练外，是否有其他调参的方法可以优化结果。

**Comment by cuicheng01** [comment]
您好，这个表格本身也属于非常困难的case了，目前的算法确实不是很好解决，需要您准备特定的数据进行标注和训练。当然，这块我们也在研发更高精度的模型，相关的模型在ready后，第一时间会release在[PaddleX](https://github.com/PaddlePaddle/PaddleX)中，敬请关注。
